### PR TITLE
Add an attribute for failure during task execution

### LINKED ISSUE #184: Missing errors from replies during proxy calls
If one of the messages fails we may want to see error details, so let's echo it in events of the `reply` method!
Right now it's hard to find why one(or more) of the messages didn't went through

### PR DESCRIPTION
**Pull request by NinoLipartiia**
Close #184

### CODE REVIEW

File: contracts/cw-croncat/src/manager.rs [hunk 1]
```diff
@@ -317,9 +317,15 @@ impl<'a> CwCroncat<'a> {
             task.interval
                 .next(&env, task.boundary, cfg.slot_granularity_time);
 
+        let response = if queue_item.failure.is_some() {
+            Response::new().add_attribute("with_failure", queue_item.failure.clone().unwrap())
```
Comment (mikedotexe) [inline contracts/cw-croncat/src/manager.rs:321]: 👍🏼

File: contracts/cw-croncat/src/tests/helpers.rs [hunk 2]
```diff
@@ -165,6 +210,97 @@ pub fn proper_instantiate() -> (App, CwTemplateContract, Addr) {
     (app, cw_template_contract, cw20_addr)
 }
 
+pub fn proper_instantiate_with_dao(
```
Comment (mikedotexe) [inline contracts/cw-croncat/src/tests/helpers.rs:213]: awesome helper function!

File: contracts/cw-croncat/src/tests/manager.rs [hunk 3]
```diff
@@ -2221,3 +2226,151 @@ fn testing_fee_works() {
...
+        for a in e.attributes {
+            if a.key == "with_failure" && a.value.contains("error executing WasmMsg") {
+                without_failure = true;
+            }
+        }
+    }
+    assert!(without_failure);
+
+    // // The proposal was not executed
+    // let res: RuleResponse<Option<Binary>> = app
```
Comment (mikedotexe) [inline contracts/cw-croncat/src/tests/manager.rs:2364]: Should we remove this?

File: contracts/cw-croncat/src/tests/manager.rs [hunk 4]
```diff
@@ -2221,3 +2226,151 @@ fn testing_fee_works() {
...
+    let proxy_call_msg = ExecuteMsg::ProxyCall { task_hash: None };
+    let res = app
+        .execute_contract(
+            Addr::unchecked(AGENT0),
+            contract_addr.clone(),
+            &proxy_call_msg,
+            &vec![],
+        )
+        .unwrap();
+    print!("{:#?}", res);
```
Comment (mikedotexe) [inline contracts/cw-croncat/src/tests/manager.rs:2350]: Great!

Screen Shot 2022-11-11 at 9 23 02 AM

**Review by mikedotexe — APPROVED** [review]

File: contracts/cw-croncat/src/tests/manager.rs (hunk 3 above)
Comment (NinoLipartiia) [reply contracts/cw-croncat/src/tests/manager.rs]: Yes, I forgot to delete it, thank you

**Pull request merged**